### PR TITLE
Added a section on Tech writing to replace the Audience section

### PR DESCRIPTION
**Pull request by dariavladykina**
A section on the nuances of technical writing to be added to the style guide for the benefit of external contributors (replace the Audience section).

### CODE REVIEW

**Comment by janajaeger** [comment]
Looks good to me. Once you have merged, maybe activate the link to the SEO section once that is merged as well.

**Review by jfaltenbacher — APPROVED** [review]
LGTM

**Review by janajaeger — APPROVED** [review]
Looks good to me. Go ahead and merge!

File: xml/docu_styleguide.techwriting.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <para>
+     Finally, ask for a linguistic review that tackles language issues, typos,
+     inconsistencies, and style guide compliance.
+    </para>
+   </listitem>
+  </varlistentry>
+ </variablelist>
+ <para>
+  <!--  For more information on how to produce meaningful content that will rank high
+  on the Web, see Section ???, “Writing for the Web”, and Section ???,
```
Comment (dariavladykina) [inline xml/docu_styleguide.techwriting.xml:74]:
```xml
see <xref linkend="sec-webwriting"/>, and <xref linkend="sec-seo"/>.
```

**Pull request merged**